README.md

```diff
@@ -172,4 +172,6 @@ Bundled assets are only generated in the remote and not merged back to this repo
 
 To deploy a demo version of the component for review, run `npm run build:demo` to create a `./build` directory that can be pushed to any static server.
 
+To run that demo, use the command: `npm run dev`.
```

Comment:
This section of the README is specific to building the project for demo deployment to a static server. This command is not appropriate for this. 

The line you are looking for us up at https://github.com/AlaskaAirlines/auro-input/pull/161/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R130